pwa/src/templates/dataLayerTemplate/DataLayerTemplate.tsx

```diff
@@ -22,9 +22,9 @@ export const ObjectsTemplate: React.FC = () => {
   return (
     <Container layoutClassName={styles.container}>
       <section className={styles.section}>
-        <Heading1>{t("Objects")}</Heading1>
+        <Heading1>{t("Data layers")}</Heading1>
         <div className={styles.buttons}>
-          <Button className={styles.buttonIcon} onClick={() => navigate(`/objects/new`)}>
+          <Button className={styles.buttonIcon} onClick={() => navigate(`/datalayers/new`)}>
```

Comment:
We don't need those backticks anymore because we dont use a variable in the string